proto/frequenz/api/common/components.proto

```diff
@@ -0,0 +1,132 @@
+// Frequenz microgrid components definitions.
+//
+// Copyright:
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
```

Comment:
Should we put 2024 as the year?

Reply:
I don't think so, this file was created in 2023 originally.